common/common.go

```diff
@@ -434,6 +443,16 @@ func writeTmpDockerfileV20180708(helper langs.LangHelper, dir string, ff *FuncFi
 		dfLines = append(dfLines, fmt.Sprintf("FROM %s", ri))
 		dfLines = append(dfLines, "WORKDIR /function")
 		dfLines = append(dfLines, helper.DockerfileCopyCmds()...)
+		
+		resources := ff.CopyResources
+
+		for _, resource := range resources {
+			if Exists(resource.From) {
+				dfLines = append(dfLines, fmt.Sprintf("COPY %s %s", resource.From, resource.To))
```

Comment:
can you quote the destination with `strconv.Quote(resource.To)` to prevent command injection here.